lib/cocoapods/external_sources/path_source.rb

```diff
@@ -27,6 +27,13 @@ def description
         "from `#{declared_path}`"
       end
 
+      # @return [Pathname] The absolute path of the podspec.
+      #
+      def podspec_path
```

Comment:
I changed this to public access with the intention of using it in the installer, but ended up changing the implementation. Will change it back